### PR TITLE
chore(examples): update README

### PR DESCRIPTION
**Pull request by leohhhn**
Updates the examples README.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
</details>

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,21 +1,37 @@
-# Gnolang examples
+# Examples
 
-This folder showcases Gnolang realms and library demos. These examples not only aid in engine testing but also provide a glimpse into the potential of Gnolang's capabilities.
+This folder showcases example Gno realms (smart contracts) and pure packages (libraries).
+These examples provide a glimpse into the potential of gno.land & Gno capabilities,
```
Comment (salmad3) [inline examples/README.md:4]:
```suggestion
These examples provide a glimpse into the potential of gno.land and the capabilities of Gno,
```

File: examples/README.md [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+while also serving as a testing set for the GnoVM.
```
Comment (salmad3) [inline examples/README.md:5]:
```suggestion
while also serving as a test suite for the GnoVM.
```

File: examples/README.md [hunk 3]
```diff
@@ -1,21 +1,37 @@
...
 
-While sharing contracts here can enhance engine testing, it's not mandatory. If considering a separate repository for contracts, be aware that this might restrict the experience due to the continuous efforts around `gno mod` support. A key point to note is that the main repository cannot reference separate code, which might pose developmental challenges.
+Pure packages and realms in this folder are pre-deployed to gno.land testnets,
+making them readily available for use on-chain. However, **there is no guarantee
```
Comment (salmad3) [inline examples/README.md:8]:
```suggestion
making them readily available for on-chain use. However, **there is no guarantee
```

File: examples/README.md [hunk 4]
```diff
@@ -1,21 +1,37 @@
...
+that code in this folder is bug-free, and should be used in accordance to this fact.**
```
Comment (salmad3) [inline examples/README.md:9]:
```suggestion
that the code is bug-free, so it should be used with caution and an understanding of potential risks.**
```

File: examples/README.md [hunk 5]
```diff
@@ -1,21 +1,37 @@
...
-
-Our recommendation is to use the [gno](../gnovm/cmd/gno) utility to develop contracts locally before publishing them on-chain. This approach offers a faster and streamlined workflow, along with additional debugging features. Simply fork or create new contracts and refer to the Makefile. Once everything looks good locally, you can then publish it on a localnet or testnet.
+## Personal Realms & Shared Content
 
-For further guidance and insights, please refer to the [`awesome-gno` tutorials](https://github.com/gnolang/awesome-gno#tutorials).
+**Prioritizing Shared Content:** As we expand our examples and use-cases, it's
+essential to prioritize shared content that benefits the broader community.
+These examples serve as a foundation and reference for all users.
+
+**Personal Realms & Pure Packages:** We're open to personal realms, but they must
```
Comment (salmad3) [inline examples/README.md:23]:
```suggestion
**Personal Realms & Pure Packages:** We welcome personal realms that
```

File: examples/README.md [hunk 6]
```diff
@@ -1,21 +1,37 @@
...
+exemplify best practices and inspire others. To maintain our repository's
```
Comment (salmad3) [inline examples/README.md:24]:
```suggestion
exemplify best practices and inspire others. To maintain the organization
```

File: examples/README.md [hunk 7]
```diff
@@ -1,21 +1,37 @@
...
+organization, we may decline some realms. If so, consider uploading
```
Comment (salmad3) [inline examples/README.md:25]:
```suggestion
of the monorepo, some submissions may be declined. If so, consider uploading
```

File: examples/README.md [hunk 8]
```diff
@@ -1,21 +1,37 @@
...
+[permissionlessly](../docs/gno-tooling/cli/gnokey/state-changing-calls.md#addpackage)
+and storing the source code on a separate repo. For higher
```
Comment (salmad3) [inline examples/README.md:27]:
```suggestion
and storing the source code in a separate repo. For higher
```

**Review by salmad3 — COMMENTED** [review]
nice info

**Review by salmad3 — APPROVED** [review]

**Review by thehowl — APPROVED** [review]

**Pull request merged**